### ISSUE TITLE
"E: Package 'vim-gnome' has no installation candidate " when installing stable Vim on Linux

### ISSUE THREAD

**Issue by rhysd** [rhysd/action-setup-vim#11]
- Job: https://github.com/rhysd/git-messenger.vim/runs/1801957774?check_suite_focus=true
- Workflow: https://github.com/rhysd/git-messenger.vim/blob/master/.github/workflows/ci.yml

```
/usr/bin/sudo apt install -y vim-gnome

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Reading package lists...
Building dependency tree...
Reading state information...
Package vim-gnome is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'vim-gnome' has no installation candidate
Error: Command 'sudo apt install -y vim-gnome' exited non-zero status 100:  WARNING: apt does not have a stable CLI interface. Use with caution in scripts.  E: Package 'vim-gnome' has no installation candidate 
Error: Command 'sudo apt install -y vim-gnome' exited non-zero status 100:  WARNING: apt does not have a stable CLI interface. Use with caution in scripts.  E: Package 'vim-gnome' has no installation candidate 
```

Related to https://github.com/rhysd/git-messenger.vim/pull/69

**Comment by rhysd** [comment]
Investigation:

- vim-gnome exists on Ubuntu 18.04 https://packages.ubuntu.com/bionic/vim-gnome
- vim-gnome does not exist on Ubuntu 20.04
- https://github.com/actions/virtual-environments#available-environments says that `ubuntu-latest` is Ubuntu 18.04
- But actually `ubuntu-latest` installs Ubuntu 20.04 https://github.com/rhysd/git-messenger.vim/runs/1801957774?check_suite_focus=true#step:1:2

**Comment by rhysd** [comment]
I could reproduce this issue with this repository's CI: https://github.com/rhysd/action-setup-vim/runs/1815005116?check_suite_focus=true

**Comment by rhysd** [comment]
Note: https://github.com/actions/virtual-environments/issues/1816

**Comment by rhysd** [comment]
Fixed in 0d17665b4a038c554347dce8a4331bc29f4c7bd1.
